Cargo.toml

```diff
@@ -1,20 +1,34 @@
-[package]
-name = "divviup-api"
+[workspace]
+members = [".", "migration", "client", "test-support", "cli"]
+
+[workspace.package]
 version = "0.0.30"
 edition = "2021"
 license = "MPL-2.0"
 homepage = "https://divviup.org"
 repository = "https://github.com/divviup/divviup-api"
+
+[workspace.dependencies]
+divviup-client = { path = "./client" }
```

Comment:
We'll still need to specify a version on these dependencies, in addition to specifying a path, in order to publish certain crates. Currently, `cargo publish --dry-run -p divviup-cli` fails because its `divviup-client` dependency doesn't specify a version.

Reply:
drat, I was hoping more of https://rust-lang.github.io/rfcs/2906-cargo-workspace-deduplicate.html had been implemented than was

Reply:
maybe we should do a publish dry run in ci?

Reply:
One barrier we've run into with Janus is that, since we're publishing two crates consecutively from the same workflow, there are some changes where the dry run for the second crate can't pass if the new version of the first crate hasn't been released.